src/cpp/core/r_util/RSourceIndex.cpp

```diff
@@ -418,6 +430,79 @@ void testThatCallIndexer(const RTokenCursor& cursor,
    }
 }
 
+void stringAfterRoxygenIndexer(const RTokenCursor& cursor,
+                               const IndexStatus& status,
+                               bool isReadOnlyFile, 
+                               RSourceIndex* pIndex)
+{
+   if (!cursor.isType(RToken::STRING))
+      return;
+   
+   RTokenCursor clone = cursor.clone();
+   if (!clone.bwdOverWhitespace())
+      return;
+
+   if (!clone.moveToPreviousToken())
+      return;   
+
+   if (!isRoxygenComment(clone))
+      return;
+
+   RSourceItem item(
+      RSourceItem::Roxygen,
+      cursor.contentAsUtf8(),
+      std::vector<RS4MethodParam>(), 
+      status.count(RToken::LBRACE),
+      cursor.row() + 1, 
+      cursor.column() + 1, 
+      isReadOnlyFile
+   );
+
+   pIndex->addSourceItem(item);
+}
+
+void nameRoxygenIndexer(const RTokenCursor& cursor,
+                        const IndexStatus& status,
+                        bool isReadOnlyFile, 
+                        RSourceIndex* pIndex)
+{
+   if (!cursor.isType(RToken::ID) || !cursor.contentEquals(L"NULL"))
+      return;
+
+   RTokenCursor clone = cursor.clone();
+   
+   if (!clone.bwdOverWhitespace())
+      return;
+
+   while (clone.moveToPreviousToken()) 
+   {
+      if (!isRoxygenComment(clone))
+         return;
+      
+      static const boost::regex nameRoxygenRegex("^#+' +(@name +.*)$");
```

Comment:
```suggestion
      static const boost::regex nameRoxygenRegex("^#+'\\s+(@name +.*)$");
```

To handle e.g. tabs.